src/components/data_comps/datm/cime_config/namelist_definition_datm.xml

```diff
@@ -453,6 +459,7 @@
       <value stream="CLMGSWP3.Solar">$DIN_LOC_ROOT_CLMFORC/atm_forcing.datm7.GSWP3.0.5d.v1.c170516/Solar3Hrly</value>
       <value stream="CLMGSWP3.Precip">$DIN_LOC_ROOT_CLMFORC/atm_forcing.datm7.GSWP3.0.5d.v1.c170516/Precip3Hrly</value>
       <value stream="CLMGSWP3.TPQW">$DIN_LOC_ROOT_CLMFORC/atm_forcing.datm7.GSWP3.0.5d.v1.c170516/TPHWL3Hrly</value>
+      <value stream="CLMNLDAS">$DIN_LOC_ROOT/atm/datm7/NLDAS</value>
```

Comment:
I took a quick look at your data. It's hourly so not dividing into streams as you have done is the right thing to do. I was going to recommend dividing it like the NLDAS2 data which is less frequent if not.

My one concern to point out is that you only have a single year of it (at least in inputdata). This will be problematic if you cycle over that single year of forcing to run a land model for multi-year climate simulations. 

It also looks like it's only for 1980, so some of the year start, end, align assignments aren't going to work right for it.

Reply:
@jonbob ^

Reply:
This configuration was first introduced some months ago by @apcraig and got lost, maybe in a CIME merge? I don't know much about it, just trying to shepherd it back in as requested.